packages/client/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@wordpress/abilities",
+  "version": "0.1.0",
+  "description": "JavaScript client for WordPress Abilities API",
+  "author": "The WordPress Contributors",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "wordpress",
+    "abilities",
+    "api",
+    "ai"
+  ],
+  "homepage": "https://github.com/WordPress/abilities-api/tree/HEAD/packages/abilities/README.md",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/WordPress/abilities-api.git",
+    "directory": "packages/abilities"
+  },
+  "bugs": {
+    "url": "https://github.com/WordPress/abilities-api/issues"
+  },
+  "engines": {
+    "node": ">=18.12.0",
+    "npm": ">=8.19.2"
+  },
+  "main": "build/index.js",
+  "module": "build-module/index.js",
+  "react-native": "src/index",
+  "wpScript": true,
+  "types": "build-types",
+  "files": [
+    "build",
+    "build-module",
+    "build-types",
+    "src",
+    "*.md"
+  ],
+  "scripts": {
```

Comment:
Is there a a reason to have these all `scripts` here instead of as part of our root `package.json`?

Reply:
Also arguably the `"workspaces": ["packages/*"]` should probably be added there too.

Reply:
Moved the main scripts to the root, added the workspace to the root. https://github.com/WordPress/abilities-api/pull/60/commits/6bb68dd96e4b9fcb605a30a6542d9cd2baf2d8cc